Wikibase/Tests/WikibaseStandardTest.php

```diff
@@ -53,6 +53,7 @@ public function testCodeSnifferStandardFiles( $sniff, $file ) {
 		$config = new Config();
 		$config->standards = [ __DIR__ . '/..' ];
 		$config->sniffs = [ $sniff ];
+		$config->reportWidth = 140;
```

Comment:
Intentional addition?

Reply:
Yes. Without this, Travis uses a default of 80, which changes all test outputs.